frdrpc/rpcserver.go

```diff
@@ -140,6 +140,42 @@ func (c *Config) wrapGetChainTransactions(ctx context.Context) func() ([]*lnrpc.
 	}
 }
 
+func (c *Config) wrapListInvoices(ctx context.Context) ([]*lnrpc.Invoice, error) {
+	resp, err := c.LightningClient.ListInvoices(ctx, &lnrpc.ListInvoiceRequest{})
```

Comment:
nit: should we timeout here?

Reply:
Added pagination + ability to cancel ctx instead?